flight/PiOS/STM32F0xx/pios_sys.c

```diff
@@ -191,6 +191,16 @@ int32_t PIOS_SYS_SerialNumberGet(char *str)
 	return 0;
 }
 
+size_t PIOS_SYS_IrqStackUnused(void)
```

Comment:
Doesn't have any code to fill the stack... and looking at startup.c it uses different symbols for irq/msp stack for some reason.